for_developers/release_checklist/index.md

```diff
@@ -85,6 +85,10 @@ This assumes you try to create `v6-22-00-patches`, adjust accordingly.
       - `make` must succeed
   1. Push to GitHub
       - `git push origin v6-22-02`
+  1. Update the stable branch. Users that have cloned this branch will receive updates as a fast-forward via `git pull`
+      - `export LATEST_STABLE=v6-xx-yy    # e.g. v6-22-02`
+      - If the stable branch does not exist yet, the `-p refs/heads/stable` part must be removed from the command below.
+      - `$ git update-ref refs/heads/stable $(git commit-tree $LATEST_STABLE^{tree} -p refs/heads/stable -m "Update stable branch to $LATEST_STABLE")`
```

Comment:
@hahnjo and I discussed about also adding a `-p $LATEST_STABLE`.  That would make the commit appear as a merge.  What do you think?  Does it improve the 'readability' of the history?

Reply:
Another thing to discuss: should the first commit in the `stable` branch have no parents (orphan branch).  This is only relevant if the user is allowed to perform any operations that involve any other branch.

Reply:
I think if we don't go for a merge commit (`-p $LATEST_STABLE^{commit}` Javier mentioned above), I'd prefer an orphan branch - I don't see a reason why the first commit would be special.